src/System.Private.ServiceModel/tools/scripts/OpenFirewallPorts.cmd

```diff
@@ -5,16 +5,32 @@ REM
 echo Start Open all ports WCF test services use
 REM
```

Comment:
This script needs to check for elevation, and bail out if we're not running elevated